### PR TITLE
Itsadssd 30861 fix consistency issues across components

### PR DESCRIPTION
**Pull request by uqtgeor4**
Done below points
**General**
- Remove old images (e.g. http://lorempixel.com/1920/1080/)
- Changed @use '@uqds/core/src/scss/global' as *; to @use '@uqds/core/src/scss/global' as core; where possible
- ::before and ::after instead of :before and :after
- "uq-" prefixing for component scss
- rem, px to its corresponding variables done for blockquote, button, form comps

**Components**
- Grid-menu: Breakpoints needs to be streamlined and scss BEM naming
- Blockquote: Convert to scss BEM naming

### CODE REVIEW

**Review by Bcrunk — CHANGES_REQUESTED** [review]
Hey Timmy 

Kitchen Sink is now missing a number of the changes you made. 

I think you still need to update the classes there?

**Review by Bcrunk — CHANGES_REQUESTED** [review]

File: packages/grid-menu/src/scss/_component.scss [hunk 1]
```diff
@@ -34,7 +33,6 @@
     @include grid-menu-4-column;
   }
 
-  a,
   &__link {
```
[the remark this answers is not in the document]
Comment (uqmchu) [reply packages/grid-menu/src/scss/_component.scss:37]: Tidied up the underline (i.e. removed the weird part underline). Hover effect (i.e. changing the background colour) can be achieved by using one of the icon pkg filter constants). There's another example of this being done on the grid-menu package SCSS file itself.

[the remark this answers is not in the document]
Comment (uqmchu) [reply packages/grid-menu/src/scss/_component.scss:37]: You can extend the icon pkg with more filter constants or just add a custom one to the grid-menu pkg with a good reason for either one.

File: packages/button/src/scss/_component.scss [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
   border: 2px solid $button-border-color;
-  border-radius: 5px;
+  border-radius: core.$space-xs;
   box-sizing: border-box;
   color: $button-color;
   cursor: pointer;
   display: inline-block;
   font-family: $button-font;
-  font-size: 1rem;
+  font-size: core.$space;
```
[the remark this answers is not in the document]
Comment (uqtgeor4) [reply packages/button/src/scss/_component.scss:14]: Done

**Review by Bcrunk — APPROVED** [review]

**Pull request merged**